packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentStatus.tsx

```diff
@@ -0,0 +1,33 @@
+import { Status, StatusProps, Typography } from '@strapi/design-system';
+
+import { capitalise } from '../../../../utils/strings';
+
+interface DocumentStatusProps
+  extends Omit<StatusProps, 'children' | 'showBullet' | 'size' | 'variant'> {
+  /**
+   * The status of the document (draft, published, etc.)
+   * @default 'draft'
+   */
+  status?: string;
```

Comment:
Can we use the status from @types here? Or is this because it needs "modified" ?

Reply:
needs to handle modified & when it's data i've found unions of strings to be really hard to manage because you have to assert it a lot e.g. we know the format of a model in the URL but we can only know its a string and then assert its the right format etc.

If it proves to be a source of bugs we can tighten it up 💪🏼